canvasapi/account.py

```diff
@@ -616,6 +616,69 @@ def list_enrollment_terms(self, **kwargs):
             **combine_kwargs(**kwargs)
         )
 
+    def list_user_logins(self, **kwargs):
+        """
+        Given a user ID, return that user's logins for the given account.
+
+        :calls: `GET /api/v1/accounts/:account_id/logins \
+        <https://canvas.instructure.com/doc/api/logins.html#method.pseudonyms.index>`_
+
+        :rtype: :class:`canvasapi.paginated_list.PaginatedList` of
+            :class:`canvasapi.login.Login`
+        """
+        from login import Login
+
+        return PaginatedList(
+            Login,
+            self._requester,
+            'GET',
+            'accounts/%s/logins' % (self.id),
+            **combine_kwargs(**kwargs)
+        )
+
+    def create_user_login(self, user, login, **kwargs):
+        """
+        Create a new login for an existing user in the given account
+
+        :calls: `POST /api/v1/accounts/:account_id/logins \
+        <https://canvas.instructure.com/doc/api/logins.html#method.pseudonyms.create>`_
+
+        :param user: The ID of the user to create the login for
+        :type user: `dict`
+        :param login: The unique ID for the new login
+        :type login: `dict`
+        :rtype: :class:`canvasapi.login.Login`
+        """
+        from login import Login
+
+        required_key_list_user = ['id']
+        required_keys_present_user = all((x in user for x in required_key_list_user))
+        required_key_list_login = ['unique_id']
+        required_keys_present_login = all((x in login for x in required_key_list_login))
```

Comment:
The `all()` syntax here is a bit excessive for only checking one key at a time. It's useful when there are a bunch of keys that need to be checked, but the `in` operator is fine for a single key.

```
required_key_list_login = ['unique_id']
required_keys_present_login = all((x in login for x in required_key_list_login))
```
is equivalent to 
```
required_keys_present_login = 'unique_id' in required_key_list_login
```

You can even take it a step further and replace the variable below with just the `in` statement, since it's only used once.